pkg/collections/transientdata/storeprovider/store/cache/cache.go

```diff
@@ -20,9 +21,11 @@ var logger = flogging.MustGetLogger("transientdata")
 
 // Cache is an in-memory key-value cache
 type Cache struct {
-	cache   gcache.Cache
-	ticker  *time.Ticker
-	dbstore transientDB
+	channelID     string
+	cache         gcache.Cache
+	ticker        *time.Ticker
+	dbstore       transientDB
+	alwaysPersist bool
```

Comment:
Do expired entries become cleared from persistence?

Reply:
Yes they do

Reply:
If the peer restarts, do we have cleanup logic to sweep expired entries from persistence?

Reply:
Yes, when the peer starts up then the sweep logic kicks in and expired entries are cleared. See the BDD test that I added in this commit to test expiration.